contrib/cirrus/setup_environment.sh

```diff
@@ -58,6 +58,9 @@ cd "${GOSRC}/"
 # Reload to incorporate any changes from above
 source "$SCRIPT_BASE/lib.sh"
 
+echo ">>>>> WORKAROUND/HACK (05/30/19): metacopy=on breaks --privileged + VFS tests"
+sed -i -r -e 's/^mountopt =.+/mountopt = "nodev"/' /etc/containers/storage.conf
```

Comment:
Didn't you mean to add `#` ?

Reply:
I'd like to stay as close as possible to the defaults, which specify nodev on all platforms.

Reply:
Okay, --force pushed, it should be happy with nodev.  If not I'll strip that too.

Reply:
I'm completely confused now. I thought the entire point of this exercise was the fact that the existence of "mountopt", no matter what the value, causes podman to die if storagedriver = vfs? 

Put another way, the above code (unless I'm missing something) does absolutely nothing, because nodev is already the default, but the default is what's breaking everything. What am I missing?

Reply:
I believe `nodev` is fine, it's what we used in F28 and F29.  F30 introduced `metacopy=on`, my understanding is that's what breaks things.  I'm totally willing to be wrong though :smile: